Source/PipeSystem/PipeSystem/Comps/CompProperties/CompProperties_ConvertThingToResource.cs

```diff
@@ -23,8 +24,10 @@ public override IEnumerable<string> ConfigErrors(ThingDef parentDef)
                 yield return "Can't use CompProperties_ConvertThingToResource with a thing that don't have Building_Storage as thingClass.";
             if (parentDef.comps.FindAll(c => c is CompProperties_ConvertThingToResource).Count > 1)
                 yield return "Can't use multiple CompProperties_ConvertThingToResource on the same thing.";
-            if (thing == null)
-                yield return "Can't use CompProperties_ConvertThingToResource with a null thing.";
+            if (thing == null & category == null)
+                yield return "Can't use CompProperties_ConvertThingToResource with a null thing or category.";
+            if (thing != null & category != null)
+                yield return "Can't use both thing and category on the same thing.";
```

Comment:
Would be easier to understand: "Can't use both thing and category on the same CompProperties_ConvertThingToResource."

Reply:
Agreed. Phrasing changed.